app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunCamera/ImageGalleryContainer/GalleryItemCard.tsx

```diff
@@ -1,32 +1,118 @@
 import { useTranslation } from 'react-i18next'
 import { useDispatch } from 'react-redux'
 
-import { StyledText } from '@opentrons/components'
-import { useHost } from '@opentrons/react-api-client'
+import { COLORS, StyledText, useCommandTextString } from '@opentrons/components'
+import {
+  useCommandQuery,
+  useDataFileRawQuery,
+  useHost,
+} from '@opentrons/react-api-client'
 
+import { Skeleton } from '/app/atoms/Skeleton'
 import { cameraPhotoOpenAction } from '/app/redux/shell'
 
 import styles from './gallery.module.css'
 
-import type { UseStubImagesInfoResult } from './hooks/useStubImagesInfo'
+import type {
+  CompletedProtocolAnalysis,
+  LabwareDefinition,
+  ProtocolAnalysisOutput,
+  RobotType,
+} from '@opentrons/shared-data'
+import type { UseImagesInfoItem } from './hooks/useImageInfo'
 
 const PHOTO_VIEWER_PADDING_PX = 16 * 2
 
-export function GalleryItemCard({
-  imagePath,
-  stepCommandText,
-  previousStepCommandText,
-  timestamp,
-}: UseStubImagesInfoResult): JSX.Element {
+interface UseImageAndCommandResult {
+  currentCommandString: string
+  previousCommandString: string
+  stubStepFraction: string
+  isLoading: boolean
+}
+export function useImage(imageId: string): string | null {
+  const { data } = useDataFileRawQuery(imageId)
+  const imagePath = data != null ? URL.createObjectURL(data) : null
```

Comment:
```suggestion
  const imagePath = null
```
For now, I think we should return `null` until we fix the photo in our follow-up. Looks like the app is crashing sometimes. Yeah this renders the skeletons 24/7, but that's ok. We're behind a feature flag anyway, which means the code is expected to be somewhat unstable to begin with.